health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/WorkflowCommentPopUp.js

```
@@ -0,0 +1,174 @@
+        // Inject the comment into the requestPayload at the given commentPath
+        const updatedPayload = setCommentInPayloadForList({ ...requestPayload }, commentPath, comment);
+
+        // Call the API using the mutation hook
+        await mutation.mutate(
+            {
+                body: updatedPayload
+            },
+            {
+                onSuccess: (data) => {
+                    setShowToast({ key: "success", label: t("WORKFLOW_UPDATE_SUCCESS") });
+                    onSuccess && onSuccess(data); // Call the onSuccess callback if provided
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)